ur_servo_control_node:
  ros__parameters:
    # Publishers
    output_servo_topic: /servo_node/delta_twist_cmds

    # Parameters
    publish_frequency: 50.0
